setup.py

```diff
@@ -42,6 +42,7 @@
     "editdistance",
     "easydict",
     "catalyst",
+    "huggingface_hub @ git+https://github.com/huggingface/huggingface_hub.git#egg=huggingface_hub" # TODO: Replace with v0.0.17 when it is released
```

Comment:
Instead of waiting for the next release of `huggingface_hub` I think we can merge this for now to start testing things on your end. I'll bump the version when it's available